Source/AwsCommonRuntimeKit/crt/CommonRuntimeError.swift

```diff
@@ -14,7 +14,7 @@ public struct CRTError: Equatable {
     public let name: String
 
     public init<T: BinaryInteger>(code: T) {
-        if code > INT32_MAX || code <= 0 {
+        if code > INT32_MAX || code < 0 {
```

Comment:
I think this was intentional. This can lead a bad call to last error becoming AWS_ERROR_SUCCESS, which will probably make the program fail at some later stage, making it harder to debug. I think it is easier to debug where the unknown error is coming from and fix that.

Reply:
Or customers will get exceptions with AWS_ERROR_SUCCESS instead of ERROR_UNKNOWN